lua/kubectl/views/containers/init.lua

```diff
@@ -22,8 +22,15 @@ function M.View(pod, ns)
 end
 
 function M.exec(pod, ns)
-  buffers.floating_buffer("k8s_container_exec", "ssh " .. M.selection)
-  commands.execute_terminal("kubectl", { "exec", "-it", pod, "-n", ns, "-c ", M.selection, "--", "/bin/sh" })
+  local cmd = "kubectl"
+  local args = string.format("exec -it %s -n %s -c %s -- /bin/sh", pod, ns, M.selection)
+
+  if config.options.terminal_cmd then
+    vim.fn.jobstart(config.options.terminal_cmd .. " " .. cmd .. " " .. args)
```

Comment:
We do some magic with the cmd inside the `commands.execute_terminal` (the `M.configure_command`), we keep it centrally since they are things we might want to do for every call (for example what context to run the command against)
Would this work for?
```lua
function M.exec(pod, ns)
  local cmd = "kubectl"
  local args = string.format("exec -it %s -n %s -c %s -- /bin/sh", pod, ns, M.selection)
  if config.options.terminal_cmd then
    cmd = config.options.terminal_cmd .. cmd
  end

  buffers.floating_buffer("k8s_container_exec", "ssh " .. M.selection)
  commands.execute_terminal(cmd, args)
end
```

Reply:
Ok added that change to the PR, worked well with both code paths.